compredict/client.py

```diff
@@ -37,12 +42,20 @@ def __init__(self,
         in dashboard)
         :param passphrase: Password to the private key.
         """
-        if token is not None and len(token) != 40:
-            raise Exception("API Key is not in valid format!")
+        self.url = api.BASE_URL.format(api.API_VERSION) if url is None else url
+        self.connection = Connection(url=self.url)
+        self.token = token
```

Comment:
I think this is redundant and not necessary.

Reply:
You mean assigning token here? I think it is necessary since it's covering the case when token is None and also username and password is None.